lib/axlsx/workbook/worksheet/worksheet.rb

```diff
@@ -686,7 +686,7 @@ def validate_sheet_name(name)
       raise ArgumentError, (ERR_SHEET_NAME_EMPTY) if name.empty?
       character_length = name.encode("utf-16")[1..-1].encode("utf-16").bytesize / 2
       raise ArgumentError, (ERR_SHEET_NAME_TOO_LONG % name) if character_length > WORKSHEET_MAX_NAME_LENGTH
-      raise ArgumentError, (ERR_SHEET_NAME_CHARACTER_FORBIDDEN % name) if '[]*/\?:'.chars.any? { |char| name.include? char }
+      raise ArgumentError, (ERR_SHEET_NAME_CHARACTER_FORBIDDEN % name) if WORKSHEET_NAME_FORBIDDEN_CHARS.chars.any? { |char| name.include? char }
```

Comment:
if we freeze as `chars` array, this can be changed to

```suggestion
      raise ArgumentError, (ERR_SHEET_NAME_CHARACTER_FORBIDDEN % name) if WORKSHEET_NAME_FORBIDDEN_CHARS.any? { |char| name.include? char }
```